src/Raven.Server/Config/Categories/SecurityConfiguration.cs

```diff
@@ -28,24 +28,24 @@ public sealed class SecurityConfiguration : ConfigurationCategory
         [ConfigurationEntry("Security.AuditLog.FolderPath", ConfigurationEntryScope.ServerWideOnly)]
         public PathSetting AuditLogPath { get; set; }
 
-        [Description("How far back we should retain audit log entries")]
-        [DefaultValue(365 * 24)]
-        [TimeUnit(TimeUnit.Hours)]
-        [ConfigurationEntry("Security.AuditLog.RetentionTimeInHours", ConfigurationEntryScope.ServerWideOnly)]
-        [ConfigurationEntry("Security.AuditLog.RetentionTimeInHrs", ConfigurationEntryScope.ServerWideOnly)]
-        public TimeSetting AuditLogRetentionTime { get; set; }
-
-        [Description("The maximum size of the audit log after which the old files will be deleted")]
-        [DefaultValue(null)]
-        [MinValue(256)]
+        [DefaultValue(128)]
+        [MinValue(16)]
         [SizeUnit(SizeUnit.Megabytes)]
-        [ConfigurationEntry("Security.AuditLog.RetentionSizeInMb", ConfigurationEntryScope.ServerWideOnly)]
-        public Size? AuditLogRetentionSize { get; set; }
+        [ConfigurationEntry("Logs.AuditLog.ArchiveAboveSizeInMb", ConfigurationEntryScope.ServerWideOnly)]
+        public Size AuditLogArchiveAboveSize { get; set; }
+
+        [DefaultValue(3)]
+        [ConfigurationEntry("Logs.AuditLog.MaxArchiveDays", ConfigurationEntryScope.ServerWideOnly)]
+        public int? AuditLogMaxArchiveDays { get; set; }
+
+        [DefaultValue(null)]
+        [MinValue(0)]
+        [ConfigurationEntry("Logs.AuditLog.MaxArchiveFiles", ConfigurationEntryScope.ServerWideOnly)]
+        public int? AuditLogMaxArchiveFiles { get; set; }
 
-        [Description("Will determine whether to compress the audit log files")]
         [DefaultValue(false)]
-        [ConfigurationEntry("Security.AuditLog.Compress", ConfigurationEntryScope.ServerWideOnly)]
-        public bool AuditLogCompress { get; set; }
+        [ConfigurationEntry("Logs.AuditLog.EnableArchiveFileCompression", ConfigurationEntryScope.ServerWideOnly)]
+        public bool AuditLogEnableArchiveFileCompression { get; set; }
```

Comment:
what about export options to another service?

Reply:
this can only be handled via config file